clients/client-python/build.gradle.kts

```diff
@@ -39,26 +39,25 @@ tasks {
   }
 
   val test by registering(VenvTask::class) {
-    dependsOn(pipInstall)
-    venvExec = "python"
-    args = listOf("-m", "unittest")
-    workingDir = projectDir.resolve(".")
-  }
-
-  val integrationTest by registering(VenvTask::class) {
-    doFirst() {
-      gravitinoServer("start")
-    }
+    val skipPyClientITs = project.hasProperty("skipPyClientITs")
+    if (!skipPyClientITs) {
+      doFirst {
+        gravitinoServer("start")
```

Comment:
Do we still need to start Gravitino manually if it is started manually in CI?

Reply:
Yes, You are right, I removed this code.